vignettes/demo.Rmd

```diff
@@ -47,9 +47,11 @@ pkg <- usethis::create_package("tea")
 ```
 
 ```{r create-package-real, echo = FALSE}
-pkg_dir <- file.path(tempdir(), "fledge")
-dir.create(pkg_dir, recursive = TRUE)
-pkg <- fledge::create_demo_project("tea", open = FALSE, dir = pkg_dir)
+parent_dir <- file.path(tempdir(), "fledge")
+dir.create(parent_dir, recursive = TRUE)
+dir.create(file.path(parent_dir, "remote"))
```

Comment:
Should we move this to the "secret" part too, and just mention that we have set up a "local" remote and in a real scenario the remote would be on GitHub? Does the output mention that we're pushing to a local remote?

Reply:
what do you mean by secret? These lines are in a chunk with `echo=FALSE`.

Reply:
I meant to refer to the code at https://github.com/cynkra/fledge/pull/164/files#diff-249dae36f675e5c47fa3956396cd295ed41b6bad97e024ace1cca47cdfa9d83cR104.